infra/notifications/src/emailSend.js

```diff
@@ -85,6 +85,16 @@ async function emailSend(
           logger.info(`No message found`)
         }
       } else {
+        const vars = {
+          dappUrl:
+            listing.id[0] === '4'
```

Comment:
This works only for staging and mainnet. Do we care about local and dev environments ? Might be nice for testing purposes to support those as well ?

Reply:
Ah yes, just added support for those. However, long term I feel that logic needs to get moved into the dapp itself: the network number is baked into the listing ID, so the dapp should do the right logic of getting the user to the right subdomain to display it. 

Likewise, the only reason we need custom logic on the ipfs gateway is because federating is currently blocked on staging. Once that is fixed we can always have the emails just point to our main gateway.